Specification/2._Basic_concepts.md

```diff
@@ -67,17 +67,17 @@ allocation procedure.
 
 ## Service invocation
 
-Service invocation is a two-step data exchange between exactly two nodes: the client and the server.
+Service invocation is a two-step data exchange between exactly two nodes: a client and a server.
 Service invocation is performed in three steps:
 
 1. The client sends a *service request* to the server.
 2. The server application takes appropriate actions and returns the response data.
```

Comment:
Indeed, the description seems confusing, but the idea is quite simple: item 2 refers to the application level, item 3 refers to the protocol level. So the response from the application is packed up and sent via UAVCAN. We can simplify things as follows:

> Service invocation is a two-step data exchange between exactly two nodes: the client and the server. Service invocation is performed as follows:
> 1. The client sends a service request to the server.
> 2. The server takes appropriate actions and sends a service response to the client.